sky/shell/platform/android/org/domokit/sky/shell/FlutterSemanticsToAndroidAccessibilityBridge.java

```diff
@@ -228,6 +228,8 @@ private void sendAccessibilityEvent(int virtualViewId, int eventType) {
             event.setPackageName(mOwner.getContext().getPackageName());
             event.setSource(mOwner, virtualViewId);
             mOwner.getParent().requestSendAccessibilityEvent(mOwner, event);
+            // TODO: You probably want to call event.setText here even though
+            // it's redundant. Other fields are needed for text editing.
```

Comment:
It's less likely to be useful for that particular event but I still think the correct behavior is just to set that field based on the node that you're firing the event on, for any event.